consensus/context/Cargo.toml

```diff
@@ -0,0 +1,24 @@
+[package]
+name = "cuprate-consensus-context"
+version = "0.1.0"
+edition = "2021"
+license = "MIT"
+authors = ["SyntheticBird","Boog900"]
+
+[dependencies]
+futures = { workspace = true, features = ["std", "async-await"] }
+tokio = { workspace = true, features = ["rt-multi-thread", "macros"]}
+tokio-util = { workspace = true }
+tower = { workspace = true, features = ["util"] }
+tracing = { workspace = true, features = ["std", "attributes"] }
+thiserror = { workspace = true }
+
+monero-serai = { workspace = true, features = ["std"] }
+randomx-rs = { workspace = true }
+rayon = { workspace = true }
+thread_local = { workspace = true }
+hex = { workspace = true }
+
+cuprate-consensus-rules =  { path = "../rules", features = ["proptest"]}
+cuprate-helper = { path = "../../helper", default-features = false, features = ["std", "cast"] }
+cuprate-types = { path = "../../types", default-features = false }
```

Comment:
can you put these crates at the top to follow how we lay this out in other crates